docs/managed-tests/08-validation.md

```diff
@@ -17,6 +20,15 @@ The following headers should not be present when submitted, and will result in a
 
 - `Woo` - as discussed [here](https://woocommerce.com/document/create-a-plugin/#section-14), this header will be added automatically during the deployment process; in order to avoid issues with incorrect headers or when distributing the same plugin outside the Marketplace, we recommend not including it manually.
 
+### Woo feature compatibility
+
+The plugin will be loaded along with WooCommerce and the WooCommerce features API will be used to check for declared incompatibility with the following features:
+
+- `custom_order_tables` - also known as HPOS; this is now the default for new WooCommerce stores, and all extensions [should be compatible with it](https://developer.woocommerce.com/docs/hpos-extension-recipe-book/#2-supporting-high-performance-order-storage-in-your-extension).
```

Comment:
Minor suggestion, I might expand this acronym to be a bit explicit:

```suggestion
- `custom_order_tables` - also known as High Performance Order Storage (HPOS); this is now the default for new WooCommerce stores, and all extensions [should be compatible with it](https://developer.woocommerce.com/docs/hpos-extension-recipe-book/#2-supporting-high-performance-order-storage-in-your-extension).
```

Reply:
Agreed, that's best practice even though HPOS should be known to extension developers.